src/main/java/ch/admin/bag/covidcertificate/sdk/core/verifier/CertificateVerifier.kt

```diff
@@ -228,11 +228,13 @@ class CertificateVerifier {
 					val nationalRulesVerifier = NationalRulesVerifier()
 					val issuedAt = certificateHolder.issuedAt?.prettyPrint(DEFAULT_DISPLAY_RULES_TIME_FORMATTER)
 					val expiredAt = certificateHolder.expirationTime?.prettyPrint(DEFAULT_DISPLAY_RULES_TIME_FORMATTER)
+					val issuer = certificateHolder.issuer
+					val kid = certificateHolder.kidBase64
```

Comment:
Same here?

Reply:
Left as `kid` as iOS does the same: https://github.com/admin-ch/CovidCertificate-SDK-iOS/pull/141/files